Tests/Form/Type/ColorSelectorTypeTest.php

```diff
@@ -268,6 +268,152 @@ public function testGetDefaultOptions()
+                'lightseagreen' => '#20B2AA',
+                'lightskyblue' => '#87CEFA',
+                'lightslategray' => '#778899',
+                'lightsteelblue' => '#B0C4DE',
+                'lightyellow' => '#FFFFE0',
+                'lime' => '#00FF00',
+                'limegreen' => '#32CD32',
+                'linen' => '#FAF0E6',
+                'maroon' => '#800000',
+                'mediumaquamarine' => '#66CDAA',
+                'mediumblue' => '#0000CD',
+                'mediumorchid' => '#BA55D3',
+                'mediumpurple' => '#9370DB',
+                'mediumseagreen' => '#3CB371',
+                'mediumslateblue' => '#7B68EE',
+                'mediumspringgreen' => '#00FA9A',
+                'mediumturquoise' => '#48D1CC',
+                'mediumvioletred' => '#C71585',
+                'midnightblue' => '#191970',
+                'mintcream' => '#F5FFFA',
+                'mistyrose' => '#FFE4E1',
+                'moccasin' => '#FFE4B5',
+                'navajowhite' => '#FFDEAD',
+                'navy' => '#000080',
+                'oldlace' => '#FDF5E6',
+                'olive' => '#808000',
+                'olivedrab' => '#6B8E23',
+                'orange' => '#FFA500',
+                'orangered' => '#FF4500',
+                'orchid' => '#DA70D6',
+                'palegoldenrod' => '#EEE8AA',
+                'palegreen' => '#98FB98',
+                'paleturquoise' => '#AFEEEE',
+                'palevioletred' => '#DB7093',
+                'papayawhip' => '#FFEFD5',
+                'peachpuff' => '#FFDAB9',
+                'peru' => '#CD853F',
+                'pink' => '#FFC0CB',
+                'plum' => '#DDA0DD',
+                'powderblue' => '#B0E0E6',
+                'purple' => '#800080',
+                'rebeccapurple' => '#663399',
+                'red' => '#FF0000',
+                'rosybrown' => '#BC8F8F',
+                'royalblue' => '#4169E1',
+                'saddlebrown' => '#8B4513',
+                'salmon' => '#FA8072',
+                'sandybrown' => '#F4A460',
+                'seagreen' => '#2E8B57',
+                'seashell' => '#FFF5EE',
+                'sienna' => '#A0522D',
+                'silver' => '#C0C0C0',
+                'skyblue' => '#87CEEB',
+                'slateblue' => '#6A5ACD',
+                'slategray' => '#708090',
+                'snow' => '#FFFAFA',
+                'springgreen' => '#00FF7F',
+                'steelblue' => '#4682B4',
+                'tan' => '#D2B48C',
+                'teal' => '#008080',
+                'thistle' => '#D8BFD8',
+                'tomato' => '#FF6347',
+                'turquoise' => '#40E0D0',
+                'violet' => '#EE82EE',
+                'wheat' => '#F5DEB3',
+                'white' => '#FFFFFF',
+                'whitesmoke' => '#F5F5F5',
+                'yellow' => '#FFFF00',
+                'yellowgreen' => '#9ACD32',
```

Comment:
How do I work with:
```bash
Remaining deprecation notices (2)

The $isInverted argument of Sonata\CoreBundle\Color\Colors::getAll is deprecated since 3.3, to be removed in 4.0: 2x
    1x in ColorSelectorTypeTest::testGetDefaultOptions from Sonata\CoreBundle\Tests\Form\Type
    1x in ColorSelectorTypeTest::testBuildForm from Sonata\CoreBundle\Tests\Form\Type
```
?